### ISSUE TITLE
Observer has already been registered

### ISSUE THREAD

**Issue by aanandshekharroy** [Anadea/RxBus#7]
This error is being thrown. Do we have any method which I can use to check before registering for events?

**Comment by dubroff** [comment]
This error signals that you did not unsubscribe properly. Check call the unregister method.

**Comment by aanandshekharroy** [comment]
Yes, I am registering in onAttach and unregistering in onDetach. Anyways,
it would have been great if there was a method like isRegistered.




Regards,
Aanand Shekhar Roy
*Freelance Android developer*
LinkedIn
<https://mailtrack.io/trace/link/781bf9092c3ab3a9d7bbc7e36dbfbae053903325?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Faanandshekharroy&userId=1101332&signature=f9ccc2653b8fba13>
 I Github
<https://mailtrack.io/trace/link/55b609e5d129de2c98ee83a0c9c4d008768be414?url=https%3A%2F%2Fgithub.com%2Faanandshekharroy&userId=1101332&signature=4179c22c484c57c8>
|
Blog
<https://mailtrack.io/trace/link/d9bea07f8a2fe83fdf879eacdb809eebf1bab21e?url=https%3A%2F%2Fmedium.com%2F%40aanandshekharroy&userId=1101332&signature=9ef73191b5f5d7a2>

**Comment by dubroff** [comment]
Ok, the proposal will be considered in the next release.
